### ISSUE TITLE
Put entrypoints in a directory

### ISSUE THREAD

**Issue by DavidBruant** [anthill/pheromon#19]
So that they're clearly identified. Also, only require `es6-shim` from these and nowhere else.

**Comment by iOiurson** [comment]
And only require `PRIVATE.json` and other conf files only in those entrypoint files.

**Comment by iOiurson** [comment]
Should the `test`entrypoint also be in this directory ? I believe not, ca you confirm @DavidBruant that here we're only talking about Api, broker and database entrypoints ?

**Comment by DavidBruant** [comment]
yeah, tests should remain in `test`. They're their own special case.
